.vscode/launch.json

```diff
@@ -21,6 +21,21 @@
       "internalDebuggerPort": 9090,
       "sourceMaps": true,
       "outDir": "${workspaceRoot}/.vscode/.react"
+    },
+    {
+      "name": "Run Tests With Debugger (slower, use npm run watch for normal work)",
+      "type": "node",
+      "request": "launch",
+      "port": 5858,
+      "address": "localhost",
+      "stopOnEntry": false,
+      "runtimeExecutable": null,
+      "runtimeArgs": [
+        "--debug-brk",
+        "./node_modules/.bin/jest",
+        "-i"
+      ],
+      "cwd": "${workspaceRoot}"
     }
```

Comment:
I was planning on adding this to vscode-jest, but for now this is a pretty good tool for poking around at the state of your tests.